### DISCUSSION TITLE
Is this project abandoned by the authors?

### DISCUSSION THREAD

**Discussion by Comonut** [sherlock-project/sherlock#1087] [General]
There has been no commits in months and I see a lot of good PRs that are ignored. 
Have the authors abdicated from maintaining this project? (not that I would blame them if they did)
Unfortunately this means that the data.json with site info is growing outdated 😢

**Comment by jcs090218** [comment]
No, they have not. I know the author @sdushantha is busy and I don't know where @TheYahya has gone yet the project is still active.

**Comment by sdushantha** [comment]
Hey @Comonut!

I appreciate you expressing your concern about Sherlock's state of development.

As @jcs090218 mentioned, I have been very busy for the past few months. I'm almost finished with my high school days so I've had to study for and write my exams. But since that is done,  I've now got to prepare for my graduation as well as plan ahead for my university days.

I've also got to update all of my other projects that have pending issues and PRs. That being said, it'll take around 2 weeks or so until I start working on Sherlock.

I plan on updating `data.json` so that Sherlock becomes more stable and hopefully returns no false positives. I'll also be creating a new branch where I'll be rewriting Sherlock from scratch. I do not want to say more about the rewrite as it is something that may only happen if time permits.

**Reply by Comonut** [reply]
I see, it's great you will be coming back to the project and good luck on your exams!

**Comment by ssjoy320** [comment]
I don't use this project

**Comment by TheYahya** [comment]
Hi @Comonut and @jcs090218, 
I've been a little busy lately, but the project haven't abandoned, I it will continue developing.

**Comment by sdushantha** [comment]
Alrighty, this project is no longer "abandoned" :)
I purged a few sites that were returning false positives and had no fix. I also fixed some sites that were fixable.
